Source/Scene/LabelCollection.js

```diff
@@ -339,13 +437,21 @@ define([
         this._scene = options.scene;
 
         this._textureAtlas = undefined;
+        this.__backgroundTextureAtlas = undefined;
+        this._whitePixelIndex = undefined;
+
+        this._backgroundBillboardCollection = new BillboardCollection({
+            scene : this._scene
+        });
+        this._backgroundBillboardCollection.destroyTextureAtlas = false;
 
         this._billboardCollection = new BillboardCollection({
             scene : this._scene
         });
         this._billboardCollection.destroyTextureAtlas = false;
 
         this._spareBillboards = [];
+        this._spareBackgroundBillboards = [];
```

Comment:
Good question.  I copied the spare glyph logic without re-thinking the reasoning why spares exist in the first place.  Does the Billboard system do a good enough job internally, destroying and creating new billboards, that it's not worth holding onto any spare billboards at all?  I'll see if I can investigate this a bit myself, but would appreciate any input from someone familiar with how billboards are destroyed and created in a collection.